numbat/src/ffi.rs

```diff
@@ -724,6 +732,23 @@ fn gamma(args: &[Value]) -> Result<Value> {
     ))))
 }
 
+fn quadratic_equation(args: &[Value]) -> Result<Value> {
```

Comment:
We can rename this to something more concise if this is a feature you want to add.

Reply:
I think `solve_quadratic_equation` would be a better name.